enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/UnionAcceptanceTest.scala

```diff
@@ -39,7 +39,7 @@ class UnionAcceptanceTest extends ExecutionEngineFunSuite with CypherComparisonS
       |M.a as B
     """.stripMargin
 
-    val result = succeedWith(Configs.Interpreted - Configs.EnterpriseInterpreted, query)
+    val result = succeedWith(Configs.CommunityInterpreted, query)
```

Comment:
Good point. I did do a tiny simplification in the Configs, but could take it further. But perhaps we do it as a Configs refactoring PR instead.